code/game/objects/explosion_recursive.dm

```diff
@@ -10,7 +10,7 @@
 var/list/explosion_turfs = list()
 var/explosion_in_progress = 0
 
-proc/explosion_rec(turf/epicenter, power)
+proc/explosion_rec(turf/epicenter, power, verbose cause = "no explanation", explosion_source = "unknown")
```

Comment:
Found why travis failed, no underscore in the two word var, will fix once on my next run.